<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortVisualizerGUIClass</class>
 <widget class="QMainWindow" name="SortVisualizerGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SortVisualizerGUI</string>
  </property>
  <widget class="QWidget" name="main_window">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="buttons">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="input">
      <item row="0" column="0">
       <widget class="QLabel" name="min_value_label">
        <property name="text">
         <string>Min value:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="min_value_input">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="max_value_lable">
        <property name="text">
         <string>Max value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="max_value_input">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="speed_label">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="speed_input">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sort_type_label">
        <property name="text">
         <string>Sort type:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="sort_type_input">
        <item>
         <property name="text">
          <string>BUBBLE_SORT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INSERTION_SORT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SELECTION_SORT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MERGE_SORT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HEAP_SORT</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="sort::SortVisualizer" name="render" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>sort::SortVisualizer</class>
   <extends>QWidget</extends>
   <header>SortVisualizer/Visualizer/SortVisualizer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SortVisualizerGUI.qrc"/>
 </resources>
 <connections/>
</ui>
